Intersect.Server/Database/Item.cs

```diff
@@ -20,44 +21,24 @@ public Item()
         {
         }
 
-        public Item(Guid itemId, int quantity, bool incStatBuffs = true) : this(
-            itemId, quantity, null, null, incStatBuffs
+        public Item(Guid itemId, int quantity, ItemProperties ItemProperties = null) : this(
+            itemId, quantity, null, null, ItemProperties
         )
         {
         }
 
-        public Item(Guid itemId, int quantity, Guid? bagId, Bag bag, bool includeStatBuffs = true)
+        public Item(Guid itemId, int quantity, Guid? bagId, Bag bag, ItemProperties itemProperties = null)
         {
             ItemId = itemId;
             Quantity = quantity;
             BagId = bagId;
             Bag = bag;
-
-            var descriptor = ItemBase.Get(ItemId);
-            if (descriptor == null || !includeStatBuffs)
-            {
-                return;
-            }
-
-            if (descriptor.ItemType != ItemTypes.Equipment)
-            {
-                return;
-            }
-
-            for (var i = 0; i < (int) Stats.StatCount; i++)
-            {
-                // TODO: What the fuck?
-                StatBuffs[i] = Randomization.Next(-descriptor.StatGrowth, descriptor.StatGrowth + 1);
-            }
+            ItemProperties = itemProperties != null ? itemProperties : new ItemProperties(ItemId);
```

Comment:
This constructor is also _not_ functionally equivalent anymore. ItemProperties are now defined with random stats 100% of the time and there's no way to turn that off.

Add `properties` between bag and includeStatBuffs, and rename includeStatBuffs to be "randomizeStatModifiers".

Stat randomization logic should remain in this constructor.